audiobooks/manifest_processing/reports/PlayBooks.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$name"        : "PlayBooks",
+    "$description" : "Google Play Books - Results from implementing an audiobook processor in the content ingestion pipeline.",
+    "a5.01"        : true,
```

Comment:
Is there a web reference to the pipeline that could be used here (using the `$href` key)? It just makes the final report sexier :-)

Reply:
The front door to the partner center is "https://play.google.com/books/publish/" -- would that be of interest?

Reply:
Sorry for the late answer; I think the answer is yes.

Reply:
No problem!  $href added.